composer.json

```diff
@@ -1,33 +1,45 @@
 {
     "name": "benjaminfavre/oauth2-http-client",
     "description": "A lightweight OAuth 2 decorator for the Symfony HTTP Client.",
-    "keywords": ["symfony", "oauth", "http-client"],
-    "homepage": "https://github.com/BenjaminFavre/oauth2-http-client",
     "license": "Apache-2.0",
+    "keywords": [
+        "symfony",
+        "oauth",
+        "http-client"
+    ],
     "authors": [
         {
             "name": "Benjamin Favre",
             "email": "favre.benjamin@gmail.com",
             "homepage": "https://github.com/BenjaminFavre"
         }
     ],
-    "autoload": {
-        "psr-4": {"BenjaminFavre\\OAuthHttpClient\\": "src/"}
-    },
+    "homepage": "https://github.com/BenjaminFavre/oauth2-http-client",
     "require": {
-        "php": "^8.0.0",
+        "php": "^8.0",
         "ext-json": "*",
-        "symfony/http-client-contracts": "^3.0",
-        "symfony/cache-contracts": "^3.0"
-    },
-    "provide": {
-        "symfony/http-client-implementation": "^2.3.1"
+        "symfony/cache-contracts": "^3.0",
+        "symfony/http-client-contracts": "^3.0"
     },
     "require-dev": {
+        "ergebnis/composer-normalize": "^2.28",
```

Comment:
Added this library to be able to normalize `composer.json`. Again it's mostly a hygiene thing. You can read about it here: https://github.com/ergebnis/composer-normalize. What I personally like about it is that it's not up to contributors or anybody else to format `composer.json` which can lead to stupid discussions and unnecessary diffs in PRs.